web/api/v1/api.go

```diff
@@ -373,6 +375,20 @@ func (api *API) queryRange(r *http.Request) (interface{}, *apiError, func()) {
 	}, nil, qry.Close
 }
 
+func (api *API) labelNames(r *http.Request) (interface{}, *apiError, func()) {
```

Comment:
Colocate this with `labelValues`?

`returnApiError` is used in `queryRange` above and is best near that. Super minor nit though.

Reply:
Done.